build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -201,13 +210,15 @@ function(iree_benchmark_suite)
       )
       if(NOT TARGET "${_TRANSLATION_TARGET_NAME}")
         add_custom_command(
-          OUTPUT "${_VMFB_FILE}"
+          OUTPUT "${_VMFB_FILE}" "${_VMFB_STAT_FILE}"
           COMMAND
             "$<TARGET_FILE:iree::tools::iree-translate>"
               ${_TRANSLATION_ARGS}
+              ${_STATS_FLAGS}
               "--mlir-print-op-on-diagnostic=false"
               "${_MODULE_SOURCE}"
               -o "${_VMFB_FILE}"
```

Comment:
This always generates stats, even if collection (`COLLECT_COMPILATION_STATISTICS`) is not enabled?
Could add a comment if that's intentional (e.g. "stats generation is cheap, so always perform it, even if not collecting the stats")